cmd/testrunner.go

```diff
@@ -1,21 +1,60 @@
 package cmd
 
 import (
-	"fmt"
-
+	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
+
+	"github.com/elastic/elastic-package/internal/packages"
+	"github.com/elastic/elastic-package/internal/testrunner/system"
 )
 
 func setupTestCommand() *cobra.Command {
-	return &cobra.Command{
+	testSystemCmd := &cobra.Command{
+		Use:   "system",
+		Short: "Run system tests",
+		Long:  "Run system tests for a package.",
+		RunE:  testSystemCommandAction,
+	}
+
+	cmd := &cobra.Command{
 		Use:   "test",
 		Short: "Run test suite for the package",
 		Long:  "Use test runners to verify if the package collects logs and metrics properly.",
 		RunE:  testCommandAction,
 	}
+	cmd.PersistentFlags().BoolP("ignoreMissing", "i", false, "ignore missing tests")
+	cmd.AddCommand(
+		testSystemCmd)
+	return cmd
 }
 
-func testCommandAction(cmd *cobra.Command, args []string) error {
-	fmt.Println("test is not implemented yet.")
+func testSystemCommandAction(cmd *cobra.Command, args []string) error {
+	packageRootPath, found, err := packages.FindPackageRoot()
+	if !found {
+		return errors.New("package root not found")
+	}
+	if err != nil {
+		return errors.Wrap(err, "locating package root failed")
+	}
+
+	ignoreMissing, err := cmd.Flags().GetBool("ignoreMissing")
+	if err != nil {
+		return errors.Wrap(err, "error parsing ignoreMissing flag")
+	}
+
+	err = system.Run(packageRootPath)
+	if err != nil && ignoreMissing && err == system.ErrNoSystemTests {
+		return nil
+	}
+
+	if err != nil {
+		return errors.Wrap(err, "error running package system tests")
+	}
+
 	return nil
 }
+
+func testCommandAction(cmd *cobra.Command, args []string) error {
```

Comment:
Follow up on another other comment:

This could be a good place to run test filtering (selected dataset) and then pass this directory to composed runners (which also can be selected with command args).